release/models/isis/openconfig-isis.yang

```diff
@@ -218,6 +224,32 @@ module openconfig-isis {
 
   // grouping statements
 
+  grouping global-counters-structural {
+    description
+      "This grouping defines global counters.";
+
+    container state {
+      config false;
+      description
+        "The container defines a list of system counters for the IS.";
+
+      uses global-counters-state;
+    }
+  }
+
+  grouping global-counters-state {
+    description
+      "This container defines global counters";
+
+    leaf total-lsps {
+      type oc-yang:counter32;
+      description
+        "A count of all LSPs in ISIS link state database across all instances
+        and VRFs.";
```

Comment:
This leaf is located within a specific IS-IS instance; does this really sum the count across _all_ instances? Or do you mean something other than routing protocol instance here?

Corollary, we have the misnomic convention of "global config" container, and repeated here as "global counters", but which are also within specific instances of the protocol. I assume this is a leftover of when we changed from a literal global IS-IS container to one under `network-instances`.

Reply:
I can see from docs that Juniper and Cisco support a per network instance counter of LSP's based on the links above.  They do not mention support for a 'global' / all instances LSP counter.  

@sulrich it is not clear what LSP counter is supported in EOS from the documentation.  Can you clarify for us?   

I am clarifying with our team what the operational use cases are for these.

Reply:
@dplore a per instance count of LSPs is available via `show isis <instance_name>
summary` command. (`LSDB Size: X`).  i'm not aware of a global multi-instance LSP counter.

additional docs: [IS-IS multiple Instances - Arista](https://www.arista.com/en/support/toi/eos-4-26-1f/14782-is-is-multiple-instances)